runtime/local/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "local-runtime"
-version = "4.39.0"
+version = "4.39.1"
```

Comment:
You need to bump the version for all these files:
* astar-collator TOML
* local-runtime TOML
* shibuya-runtime TOML
* shiden-runtime TOML

They should **all** be the same.

Additionally, since you're introducing new features, you should bump the minor version.